nbs/01_io.ipynb

```diff
@@ -266,6 +268,151 @@
     "    return isinstance(s_rep,list) or isinstance(s_rep,np.ndarray)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| export\n",
+    "def create_new_columns_and_meta(df,structure):\n",
+    "    dfs_new, metas_new = [], []\n",
+    "    for dict_with_create in filter(lambda s: 'create' in s.keys(), structure):\n",
+    "        if dict_with_create['create']['type'] == 'topk':\n",
```

Comment:
I would have a dict with the functions and check the dict. I.e. above define a global dict
`create_block_types = {'topk': add_topk_structure(df,dict_with_create) }`
and then see if the type is in that dict and if is, call the appropriate function. 
Avoids creating an if-statement every time a new block type gets added.